pkgs/applications/editors/emacs/generic.nix

```diff
@@ -67,18 +67,18 @@ in stdenv.mkDerivation {
 
     # Make native compilation work both inside and outside of nix build
     (lib.optionalString nativeComp (let
-      libPath = lib.concatStringsSep ":" [
-        "${lib.getLib libgccjit}/lib/gcc/${targetPlatform.config}/${libgccjit.version}"
-        "${lib.getLib stdenv.cc.cc}/lib"
-        "${lib.getLib stdenv.glibc}/lib"
-      ];
+      libPath = (lib.concatStringsSep " "
+        (builtins.map (x: ''\"-L${x}\"'') [
+          "${lib.getLib libgccjit}/lib"
+          "${lib.getLib libgccjit}/lib/gcc/${targetPlatform.config}/${libgccjit.version}"
+          "${lib.getLib stdenv.cc.cc}/lib"
+          "${lib.getLib stdenv.libc}/lib"
```

Comment:
Should this be glibc? 

```
nix-repl> :l <nixpkgs>
[0.0 MiB DL]Added 12314 variables.

nix-repl> lib.getLib stdenv.libc
error: attribute 'libc' missing, at (string):1:12
```

Reply:
Ah, so that should be a conditional! Good to know. Will correct.

Reply:
`stdenv.cc.libc` is the portable way to do this.